src/main/java/games/strategy/triplea/util/AiBonusIncomeUtils.java

```diff
@@ -0,0 +1,38 @@
+package games.strategy.triplea.util;
```

Comment:
Just curious why you chose to extract this class to this package instead of to `g.s.triplea.ai`, where there are other AI-related `*Utils` classes?  Is it because the utilities in the `g.s.triplea.ai` package are intended for use by AI implementations, while this class is intended for general use?  I seem to remember a PR a few months ago where there was some excitement over non-AI code referencing utility code under `g.s.triplea.ai`.